docs/src/pages/components/popper/VirtualElementPopper.tsx

```diff
@@ -13,9 +13,9 @@ const useStyles = makeStyles((theme: Theme) =>
   }),
 );
 
-export default function FakedReferencePopper() {
+export default function VirtualElementPopper() {
   const [open, setOpen] = React.useState(false);
-  const [anchorEl, setAnchorEl] = React.useState<PopperProps['anchorEl']>(null);
+  const [anchorEl, setAnchorEl] = React.useState<PopperProps.anchorEl>(null);
```

Comment:
For some reason, eslint always changes this for me...

Reply:
eslint shouldn't touch this file to begin with. What IDE + extension are you using to run eslint on this file?

Reply:
Ah looks like it's a webstorm feature:

> When checking if ESLint should be run on a .ts files, WebStorm 2018.3 analyzes the root config statically to see if it includes the following plugins:
> 
> "parser":"babel-eslint" or
> "parser":"typescript-eslint-parser" or
> "eslint-plugin-typescript" or
> "@typescript-eslint/parser"

I can turn it off by adding typescript files to .eslintignore

Reply:
Can't you tell webstorm to ignore it? Though it might resolve itself after #21758